src/main/scala/org/broadinstitute/dsde/firecloud/service/NihService.scala

```diff
@@ -167,7 +167,7 @@ class NihService(val samDao: SamDAO, val thurloeDao: ThurloeDAO, val googleDao:
     ecmDao.putLinkedEraAccount(LinkedEraAccount(userInfo.id, nihLink))(getAdminAccessToken)
     .flatMap(_ => {
       logger.info("Successfully linked NIH account in ECM for user " + userInfo.id)
-      Future.successful(Success())
+      Future.successful(Success(()))
```

Comment:
this addresses a compile warning:
```
[warn] /Users/davidan/work/src/firecloud-orchestration/src/main/scala/org/broadinstitute/dsde/firecloud/service/NihService.scala:170:32: adaptation of an empty argument list by inserting () is deprecated: this is unlikely to be what you want
[warn]         signature: Success.apply[T](value: T): scala.util.Success[T]
[warn]   given arguments: <none>
[warn]  after adaptation: Success((): Unit)
[warn]       Future.successful(Success())
[warn]                                ^
```
there are many instances of this fix in this PR